### ISSUE TITLE
Original Games do not work anymore after installing Retroarch 1.0

### ISSUE THREAD

**Issue by Evilmachine** [ClusterM/retroarch-clover#109]
Hi there,

the SNES Mini Original Games do not work after installing Retroarh 1.0.

Even the Covers do not load. The not original games work by using the embedded emulator and retroarch.

**Comment by TheDribble** [comment]
I had this issue but restoring the original core then re-uploading the Games and finally uploading the custom core fixed it

**Comment by DecanFrost** [comment]
can you elaborate pls..   Original Core ?? as in Original SNES Kernel, or Retroach core (older version)?

**Comment by TheDribble** [comment]
Original snes kernel

**Comment by DecanFrost** [comment]
Ok im a littel confused..  sorry,  but after flashing the orignal Kernel back, you cant upload the games unless you flash the custom kernel first.. but you describe it the other way,  STOCK -- then Games -- then Custom

**Comment by TheDribble** [comment]
correct, sorry for the confusion

**Comment by TheDribble** [comment]
that is what fixed it for me

**Comment by TheDribble** [comment]
so when you try to add the game it will autoflash the custom kernal then add the mods afterwards like retroarch

**Comment by DecanFrost** [comment]
Gave it a try,  Flashed Stock,  Uploaded games, it asked to flash custom and then it uploaded the games automatically and then i flashed retroarch 1.0   still some of the original games not loading anymore

**Comment by Pito23** [comment]
yes i've some issues like yours after every hakchi or retroarch updates... some games doesn't launch anymore or for some others issues like corrupted icon and so on... i've the portable version...

**Comment by Pito23** [comment]
i forgot to mention that, the only way to fix them is uninstall hakchi, flash original kernel, reflash custom kernel and reinstall everything.... very annoying...

**Comment by troytroy22** [comment]
all i did to fix mine was to reinstall the custom kernal and my original games worked. 1.0c still broke my original games...
